### DISCUSSION TITLE
Supported (cheap) USB fan hubs / controller

### DISCUSSION THREAD

**Discussion by JanFschr** [Rem0o/FanControl.Releases#1246] [Q&A]
Hello, my motherboards sensors or better the fan control is not supported. So im looking for a cheap usb controller to control my chassis fans. any suggestions whats supported by fancontrol out of the box?

**Comment by Rem0o** [comment]
There is no built-in controller that will work out the box. Some people developed basic plugins to support their controllers, but nothing I can guarantee.

Some companies, some small, some surprisingly big, have contacted me and expressed their desire to collab/offer a plugin for their case/controllers.

Like ElmorLabs, which I must shout-out since they seem to have a very open attitude by providing SDK-style software to control their hardware (https://github.com/ElmorLabs), something I wish ALL hardware manufacturer would do: 

https://www.elmorlabs.com/product/elmorlabs-efc-x9-easy-fan-controller-for-9-fans/
https://github.com/ElmorLabs/FanControl.EFC_X9
